app/javascript/components/Header/index.js

```diff
@@ -1,110 +1,55 @@
-      .then(_ => setOfficeValue(office.id))
-      .then(_ => togglePopover('user'))
-
-  return (
-    <div className={s.header}>
-      <div className={s.container}>
-        <div className={adminPage ? `${s.wrapper} ${s.wrapperAdmin}` : s.wrapper}>
-          <div className={s.logoBox}>
-            <Link to="/portal">
-              <img
-                alt="Zendesk Relationshapes Logo"
-                className={s.logo}
-                src="//d1eipm3vz40hy0.cloudfront.net/images/part-header/zendesk-relationshapes-logo.svg"
-              />
-            </Link>
-          </div>
-          <AdminActions currentUser={currentUser} />
-          <div>
-            <button className={s.btn} onClick={togglePopover}>
-              <img className={s.img} src={currentUser.photo} />
-              <div className={s.userInfoBox}>
-                <div className={s.userName}>{currentUser.name}</div>
-                <div className={s.userOffice}>{currentUser.office && currentUser.office.name}</div>
-              </div>
-              <Popover
-                className={s.popover}
-                open={present(popover)}
-                anchorEl={popover ? popover.anchorEl : null}
-                anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}
-                targetOrigin={{ horizontal: 'right', vertical: 'top' }}
-                onRequestClose={togglePopover}
-              >
-                <Menu>
-                  <MenuItem
-                    style={styles.item}
-                    primaryText="Default Office"
-                    rightIcon={<ArrowDropRight />}
-                    menuItems={offices.map((o, i) => (
-                      <MenuItem
-                        key={`office-${i}`}
-                        primaryText={o.name}
-                        style={styles.item}
-                        checked={currentUser.office.id === o.id}
-                        onClick={() => handleOfficeSelect(o)}
-                        insetChildren
-                      />
-                    ))}
-                  />
-                  <Divider />
-                  <a href="/users/sign_out">
-                    <MenuItem style={styles.item} primaryText="Sign out" />
-                  </a>
-                </Menu>
-              </Popover>
-            </button>
-          </div>
-        </div>
-      </div>
-      <div style={{ overflow: 'auto' }} />
-    </div>
-  )
-}
+import styled from 'styled-components'
+
+const Container = styled.div`
+  display: flex;
+  width: 100%;
+  height: 100px;
+  margin: 0 0 50px 0;
+  padding: 0;
+  box-shadow: 0px 0px 5px #bbb;
+  justify-content: space-around;
+`
+
+const Wrapper = styled.div`
+  padding: 0 20px;
+  width: 935px;
+  display: flex;
+  justify-content: flex-start;
+  flex-flow: row nowrap;
+  align-items: center;
+`
+
+const Logo = styled.img`
+  height: 40px;
+`
+
+const LogoBox = styled.div`
+  flex-grow: 2;
+`
+
+const LogoSection = () => (
```

Comment:
this is not being used anywhere else but inside `Header`, I would just embed it inside tbh.

Reply:
good point!